packages/pointer_interceptor/pointer_interceptor/example/integration_test/widget_test.dart

```diff
@@ -4,176 +4,13 @@
-        print('Skipping test: Needs flutter > 2.10');
-        return;
-      }
-
-      final html.Element element =
-          _getHtmlElementAtCenter(nonClickableButtonFinder, tester);
-
-      expect(element.tagName.toLowerCase(), 'flt-semantics');
-      expect(element.getAttribute('aria-label'), 'Never calls onPressed');
-    });
-
-    // Notice that, when hit-testing the background platform view, instead of
-    // finding a semantics node, the platform view itself is found. This is
-    // because the platform view does not add interactive semantics nodes into
-    // the framework's semantics tree. Instead, its semantics is determined by
-    // the HTML content of the platform view itself. Flutter's semantics tree
-    // simply allows the hit test to land on the platform view by making itself
-    // hit test transparent.
-    testWidgets('on background directly', (WidgetTester tester) async {
-      app.main();
-      await tester.pumpAndSettle();
-
-      final html.Element element =
-          _getHtmlElementAt(tester.getTopLeft(backgroundFinder));
-
-      expect(element.id, 'background-html-view');
-    });
-  });
-}
-
-// Calls [_getHtmlElementAt] passing it the center of the widget identified by
-// the `finder`.
-html.Element _getHtmlElementAtCenter(Finder finder, WidgetTester tester) {
-  final Offset point = tester.getCenter(finder);
-  return _getHtmlElementAt(point);
-}
-
-// Locates the DOM element at the given `point` using `elementFromPoint`.
-//
-// `elementFromPoint` is an approximate proxy for a hit test, although it's
-// sensitive to the presence of shadow roots and browser quirks (not all
-// browsers agree on what it should return in all situations). Since this test
-// runs only in Chromium, it relies on Chromium's behavior.
-html.Element _getHtmlElementAt(Offset point) {
-  // Probe at the shadow so the browser reports semantics nodes in addition to
-  // platform view elements. If probed from `html.document` the browser hides
-  // the contents of <flt-glass-name> as an implementation detail.
-  final html.ShadowRoot glassPaneShadow =
-      html.document.querySelector('flt-glass-pane')!.shadowRoot!;
-  return glassPaneShadow.elementFromPoint(point.dx.toInt(), point.dy.toInt())!;
-}
-
-// TODO(dit): Remove this after flutter master (2.13) lands into stable.
-// This detects that we can do new semantics assertions by looking at the 'id'
-// attribute on flt-semantics elements (it is now set in 2.13 and up).
-bool _newSemanticsAvailable() {
-  final html.ShadowRoot glassPaneShadow =
-      html.document.querySelector('flt-glass-pane')!.shadowRoot!;
-  final List<html.Element> elements =
-      glassPaneShadow.querySelectorAll('flt-semantics[id]');
-  return elements.isNotEmpty;
+  // TODO(louisehsu): given the difficulty of making the same integration tests
+  // work for both web and ios implementations, please find tests in their respective
+  // platform implementation packages.
```

Comment:
If there are not going to be tests here, I'd fully remove `integration_test` from the `example` app. I don't think it'll annoy the CI too much.

You already have the `tests_exist_elsewhere_test.dart` in the main directory, so as long as you point people to the implementation-specific tests from there, I think you can remove the ones here altogether!